### PR TITLE
Update wear compose scrollaway with timeout changes

### PR DESCRIPTION
**Pull request by rajat4064g**
Adding animations to scrollAway modifier.

TimeText will appear after 3s when scrolling stops in a list of items.

### CODE REVIEW

File: compose-layout/src/main/java/com/google/android/horologist/compose/layout/ScaffoldState.kt [hunk 1]
```diff
@@ -50,7 +50,7 @@ internal class ScaffoldState {
             Box(
                 modifier = Modifier
                     .fillMaxSize()
-                    .scrollAway {
+                    .scrollAway(true) {
```
Comment (yschimke) [inline compose-layout/src/main/java/com/google/android/horologist/compose/layout/ScaffoldState.kt:53]: Can we use named params?  

Is this meant to be defaulted on? Would false be a better default? Or move this up to a param for AppScaffold/ScreenScaffold?

**Review by yschimke — APPROVED** [review]
LGTM - but I don't think we should change the default at least yet.

**Comment by yschimke** [comment]
@rajat4064g let me know if you are waiting on me for anything.

**Comment by rajat4064g** [comment]
> @rajat4064g let me know if you are waiting on me for anything.

There are more changes that need to be dropped in this branch regarding syncing the timer for TimeOut animations between material PositionIndicator and new ScrollAway. Will merge this PR when that is finalised.

**Comment by yschimke** [comment]
Going to close as it's got merge conflicts, reopen once it's ready for review.

**Pull request closed without merging**